forc-pkg/src/lock.rs

```diff
@@ -296,30 +296,25 @@ fn pkg_dep_line(
 type ParsedPkgLine<'a> = (Option<fuel_tx::Salt>, Option<&'a str>, &'a str);
 // Parse the given `PkgDepLine` into its dependency name and unique string segments.
 //
-// I.e. given "(<salt>) (<dep_name>) <name> <source>", returns ("<salt>", "<dep_name>", "<name> <source>").
+// I.e. given "(<dep_name>) <name> <source> (<salt>)", returns ("<salt>", "<dep_name>", "<name> <source>").
```

Comment:
Perhaps we should change the order of the returned tuple to match the order of the input, e.g.
```suggestion
// I.e. given "(<dep_name>) <name> <source> (<salt>)", returns ("<dep_name>", "<name> <source>", "<salt>").
```